java/client/src/main/java/glide/api/RedisClient.java

```diff
@@ -78,4 +87,118 @@ public CompletableFuture<Object> customCommand(String[] args) {
                 Command.builder().requestType(Command.RequestType.CUSTOM_COMMAND).arguments(args).build();
         return commandManager.submitNewCommand(command, BaseClient::handleObjectResponse);
     }
+
+    /**
+     * Ping the Redis server.
+     *
+     * @see <a href="https://redis.io/commands/ping/">redis.io</a> for details.
+     * @returns the String "PONG"
+     */
+    @Override
+    public CompletableFuture<String> ping() {
+        Command command = Command.builder().requestType(Command.RequestType.PING).build();
+        return commandManager.submitNewCommand(command, BaseClient::handleStringResponse);
+    }
+
+    /**
+     * Ping the Redis server.
+     *
+     * @see <a href="https://redis.io/commands/ping/">redis.io</a> for details.
+     * @param msg - the ping argument that will be returned.
+     * @returns return a copy of the argument.
+     */
+    @Override
+    public CompletableFuture<String> ping(String msg) {
+        Command command =
+                Command.builder()
+                        .requestType(Command.RequestType.PING)
+                        .arguments(new String[] {msg})
+                        .build();
+        return commandManager.submitNewCommand(command, BaseClient::handleStringResponse);
+    }
+
+    /**
+     * Get information and statistics about the Redis server. DEFAULT option is assumed
+     *
+     * @see <a href="https://redis.io/commands/info/">redis.io</a> for details.
+     * @return CompletableFuture with the response
+     */
+    @Override
+    public CompletableFuture<Map> info() {
+        Command command = Command.builder().requestType(Command.RequestType.INFO).build();
+        return commandManager.submitNewCommand(command, BaseClient::handleMapResponse);
+    }
+
+    /**
+     * Get information and statistics about the Redis server.
+     *
+     * @see <a href="https://redis.io/commands/info/">redis.io</a> for details.
+     * @param options - A list of InfoSection values specifying which sections of information to
+     *     retrieve. When no parameter is provided, the default option is assumed.
+     * @return CompletableFuture with the response
+     */
+    @Override
+    public CompletableFuture<Map> info(InfoOptions options) {
+        Command command =
+                Command.builder()
+                        .requestType(Command.RequestType.INFO)
+                        .arguments(options.toInfoOptions())
+                        .build();
+        return commandManager.submitNewCommand(command, BaseClient::handleMapResponse);
+    }
+
+    /**
+     * Get the value associated with the given key, or null if no such value exists.
+     *
+     * @see <a href="https://redis.io/commands/get/">redis.io</a> for details.
+     * @param key - The key to retrieve from the database.
+     * @return If `key` exists, returns the value of `key` as a string. Otherwise, return null
```

Comment:
1. dashes
2. too many returns
```suggestion
     * @param key The key to retrieve from the database.
     * @return A value of `key` as a string if it exists, null otherwise.
```